src/basic/chase-symlinks.h

```diff
@@ -23,7 +23,7 @@ bool unsafe_transition(const struct stat *a, const struct stat *b);
 /* How many iterations to execute before returning -ELOOP */
 #define CHASE_SYMLINKS_MAX 32
 
-int chase_symlinks(const char *path_with_prefix, const char *root, ChaseSymlinksFlags chase_flags, char **ret_path, int *ret_fd);
```

Comment:
the function carried this name because unless you specify CHASE_PREFIX_ROOT it must be the path with the root prefix already prepended, which is noteworthy, because I guess people in many contexts would not expect that.